### ISSUE TITLE
Enhance getSettlementById endpoint to report window content

### ISSUE THREAD

**Issue by ggrg** [mojaloop/project#1208]
## **Goal**:
As a Hub Operator
I want to view the windows content when I query GET /settlements/{id} endpoint
so that response also reports windows content

**Tasks**:
- [x] Modify DAO script (@bothadeon)
- [x] Modify handler to group db query result and response (@bothadeon)
- [x] Update Integration tests (@bothadeon)
- [x] Add unit tests to cover delivered functionality (@bothadeon)
- [x] Pull request (@bothadeon)

**Acceptance Criteria**:
- [x] Designs are up-to date
- [x] Unit Tests pass
- [x] Swagger definition is up to date
- [x] Integration Tests pass
- [x] Code Style & Coverage meets standards
- [x] Changes made to config (default.json) are broadcast to team and follow-up tasks added to update helm charts and other deployment config.
- [x] GET /settlements/{id} endpoint functionality is enhanced to report Settlements windows content as well

## **Pull Requests**:
- [ ] Pull request : https://github.com/mojaloop/central-settlement/pull/263

## **Follow-up**:
- N/A

**Dependencies**:
- dependent on story #1175.

## **Accountability**:
- Owner: @ggrg
- QA/Review: @ggrg , @ndonnan

**Comment by bothadeon** [comment]
Pull request created : https://github.com/mojaloop/central-settlement/pull/241

**Comment by bothadeon** [comment]
Story is blocked due to a dependancy on : story #1175.

**Comment by bothadeon** [comment]
Created a new pull request after local branch got corrupted : https://github.com/mojaloop/central-settlement/pull/255

**Comment by bothadeon** [comment]
Added the filter on get settlements window content by settlement id and settlement window id,
Refer to pull request : https://github.com/mojaloop/central-settlement/pull/263
